platforms/core-runtime/io/src/main/java/org/gradle/internal/io/LinePerThreadBufferingOutputStream.java

```diff
@@ -16,16 +16,15 @@
 
 package org.gradle.internal.io;
 
-import org.gradle.internal.SystemProperties;
-
 import java.io.IOException;
 import java.io.PrintStream;
 import java.util.Locale;
 
 public class LinePerThreadBufferingOutputStream extends PrintStream {
     private final TextStream handler;
+    @SuppressWarnings("ThreadLocalUsage")
     private final ThreadLocal<PrintStream> stream = new ThreadLocal<PrintStream>();
-    private final String lineSeparator = SystemProperties.getInstance().getLineSeparator();
+    private final String lineSeparator = System.getProperty("line.separator");
```

Comment:
💅  Can this be replaced with `System.lineSeparator()` or it's not available on that language level?

Reply:
We need to be Java 6 compatible here as this code is potentially used in workers.